buildenv/jenkins/openjdk10_s390x_linux

```diff
@@ -3,7 +3,7 @@
 	Set parameter TARGET(openjdk, system, external, perf, jck etc.).
 	Set UPSTREAM_JOB_NAME(version_build_arch_os, for example: openjdk8_build_x86-64_linux)
 	Set JVM_VERSION(openjdk8, openjdk8-openj9, openjdk9, openjdk8-openj9, etc.*/
-LABEL='linux&&s390x&&test'
+LABEL='linux&&s390x&&test||ci.project.openj9&&sw.os.linux&&hw.arch.s390'
 
```

Comment:
could become: sw.os.linux&&hw.arch.s390

Reply:
https://ci.adoptopenjdk.net/computer/test-marist-ubuntu1604-s390x-1/  s390 label has not been updated.

Reply:
Updated now

Reply:
updated.